reference.graphql

```diff
@@ -693,10 +693,10 @@ type Market implements Pool @entity {
   maximumLTV: BigDecimal!
 
   " Liquidation threshold as a percentage value "
-  liquidationThreshold: BigDecimal!
+  liquidateThreshold: BigDecimal!
```

Comment:
you can leave `liquidateThreshold` and `liquidatePenalty`

Reply:
done